dev/chart/build_changelog_annotations.py

```diff
@@ -98,7 +98,7 @@ def print_entry(section: str, description: str, pr_number: int | None):
             in_first_release = True
         # Make sure we get past "significant features" before we actually start keeping track
         elif not past_significant_changes:
-            if line == "New Features":
+            if line in TYPE_MAPPING:
```

Comment:
The TYPE_MAPPING has 5 key-value pairs in it.

So this change might have a different result from the previous code which only checks if the line equals "New Features". Could you explain this change, please?

Reply:
It's quite obvious - the code will fail if there are no New Features (like in 1.13.1